app/channels/application_cable/channel.rb

```diff
@@ -1,4 +1,5 @@
 module ApplicationCable
+  # default channel
```

Comment:
@marderer do you like such comments?

Reply:
`ApplicationCable` is default cable, and its channel is default channel. No need in comment

Reply:
@AleksSenkou i dot't know how disable it in reek

Reply:
@marderer can you try to ask help from other students? As I noticed from review - part of guys disabled those comments

Reply:
@AleksSenkou Yes, of course